src/StoreKeeper/WooCommerce/B2C/Endpoints/TaskProcessor/TaskProcessorEndpoint.php

```diff
@@ -36,4 +36,14 @@ public function handle()
 
         return $response;
     }
+
+    final public function handleRequestSilently(\WP_REST_Request $request): ?\WP_REST_Response
+    {
+        $response = $this->handleRequest($request);
+        if (500 === $response->get_status()) {
+            return $response;
+        }
+
+        return null;
```

Comment:
can you also add the `-s` option to the curl itself?

```
curl --help
 -s, --silent        Silent mode
```

Reply:
Added on `SchedulerTab`.